Drink_Book_PR/DrinkBook/User.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DrinkBook
+{
+    public class User
+    {
+        private string name;
+        private int age;
+        private int budget;
+        private int whiskeysToDrink;
+        private int vodkasToDrink;
+        private string preferredMusic;
+
+        public string Name
```

Comment:
Suggestion You can make use of the Auto-Implemented properties. Ref: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties